### PR TITLE
Create Add Button and Popover

### PR DESCRIPTION
**Pull request by Jdellis818**
This adds the structure/ui for adding food entries and tags. I know some of the styling stuff isn't ideal, but I was getting really frustrated with it so I just went with what worked. Functionality to upload a photo still needs to be added at some point.

### CODE REVIEW

File: src/app/components/add-food-tab/add-food-tab.component.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.form-field {
+  display: block;
+  width: 100%;
+}
+
+.rating {
+  outline: none;
+  font-size: 30pt;
```
Comment (charkour) [inline src/app/components/add-food-tab/add-food-tab.component.scss]: I have a feeling Norman might make us down for not using relative styles.

File: src/styles.scss [hunk 2]
```diff
@@ -9,3 +9,14 @@ body {
...
+.mat-tab-body-content {
+  overflow: hidden !important;
+}
+
+button:focus {
+  outline: none !important;
+}
+
+/* Importing Bootstrap SCSS file. */
+@import "~bootstrap/dist/css/bootstrap";
```
Comment (charkour) [inline src/styles.scss]: Typically, you would want to put this at the top of the file.

**Review by charkour — APPROVED** [review]
LGTM. I'll let you merge it since there are two minor comments. You can see if it is worth it to change it or not.

File: src/styles.scss (hunk 2 above)
Comment (Jdellis818) [reply src/styles.scss]: I'm not sure why, but this is where it was added automatically with ng add

**Pull request merged**